torch/csrc/api/include/torch/optim/optimizer.h

```diff
@@ -91,9 +91,7 @@ class TORCH_API OptimizerParamGroup {
   const std::vector<Tensor>& params() const;
 
  protected:
-  // NOLINTNEXTLINE(cppcoreguidelines-non-private-member-variables-in-classes)
```

Comment:
huh, these NOLINT look appropriate, the header is just not on the current include path.

Reply:
The reason is that we have put '-cppcoreguidelines-non-private-member-variables-in-classes' in the project clang-tidy config file. When I changed to those files, I would like to remove them. If you think we should put them back, I will revert the NOLINTs